.ghci

```diff
@@ -1 +1,31 @@
 :set -isrc
+
+bind 0 x: {v: int | 0 < v}
```

Comment:
Haven't the slightest idea --- didn't even know that file existed!

> On Apr 19, 2016, at 6:35 PM, Eric Seidel <notifications@github.com> wrote:
> 
> In .ghci:
> 
> > @@ -1 +1,31 @@
> >  :set -isrc
> > +
> > +bind 0 x: {v: int | 0 < v}
> What's this about? Bad paste?
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
>